x/margin/types/genesis.go

```diff
@@ -20,6 +20,7 @@ func DefaultGenesis() *GenesisState {
 			EpochLength:           1,
 			MaxOpenPositions:      10000,
 			Pools:                 []string{},
+			SqModifier:            sdk.MustNewDecFromStr("10000000000000000000000000"),
```

Comment:
@sifag do we have a spec somewhere about this default value? thanks

Reply:
If you look at the diff, it was the default value that was set

Reply:
@caner this default value still needs to be decided.